api/middleware/inbound_test.go

```diff
@@ -26,13 +26,15 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/golang/mock/gomock"
-	"github.com/stretchr/testify/assert"
 	"go.uber.org/yarpc/api/middleware"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/api/transport/transporttest"
 	"go.uber.org/yarpc/encoding/raw"
 	"go.uber.org/yarpc/internal/testtime"
+
```

Comment:
No newline

Reply:
Are we not dividing imports? I thought we followed the pattern of:
```
... go library imports ...

... project imports ...

... external imports ...
```

Reply:
@AlexAPB has the right of it here I believe :)

Reply:
I thought we agreed for yarpc-go we weren't doing this anymore @abhinav